src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -181,13 +194,13 @@ void EnIk_SetupAction(EnIk* this, EnIkActionFunc actionFunc) {
     this->actionFunc = actionFunc;
 }
 
-void func_80A74398(Actor* thisx, PlayState* play) {
+void EnIk_MoreInit(Actor* thisx, PlayState* play) {
```

Comment:
```suggestion
void EnIk_InitImpl(Actor* thisx, PlayState* play) {
```
not that I disagree with "MoreInit" or the sentiment behind it lol

Reply:
think my thought process was like, Impl (to me) is usually when all of the logic is delegated to that function, and the thing that calls it is just a wrapper. Where in this case the actual init func does a few things, then *most* of the rest is in the this function. idk

I can change it to impl as well, cause I again i dont care that much